clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -953,6 +961,21 @@ private synchronized boolean threadShouldExit(long now, long curHardShutdownTime
             return false;
         }
 
+        private void failPendingCalls(long now, Map<Node, List<Call>> callsToSend, Throwable t) {
+            synchronized (this) {
+                for (Call call : newCalls) {
+                    call.fail(now, t);
+                }
+                newCalls.clear();
+            }
+            for (List<Call> calls : callsToSend.values()) {
```

Comment:
Probably add comment on why synchronization on callsToSend is not needed.

Reply:
I will add a comment to the declaration of callsToSend, since this is used in many other places.